demos/background_subtraction_demo/cpp_gapi/background_subtraction_demo_gapi.hpp

```diff
@@ -47,6 +58,9 @@ DEFINE_bool(no_show, false, no_show_message);
 DEFINE_string(target_bgr, "", target_bgr_message);
 DEFINE_uint32(blur_bgr, 0, blur_bgr_message);
 DEFINE_string(u, "", utilization_monitors_message);
+DEFINE_bool(use_onevpl, false, use_onevpl_message);
+DEFINE_string(onevpl_params, "", onevpl_params_message);
+DEFINE_uint32(onevpl_pool_size, 0, onevpl_pool_size_message);
```

Comment:
```suggestion
DEFINE_uint32(onevpl_pool_size, 25, onevpl_pool_size_message);
```

Reply:
This parameter doesn't have a `god` value. It must be adjusted for specific execution (video, model ...)

Reply:
Suggest leave zero as default

Reply:
Can you add the last sentence to a flag description?)
Also as I understand the value depends on a system as well. Because for me the smallest possible value is 12. Could you add the word system to `(video, model ...)`

Reply:
Done